modules/nf-core/cellranger/vdj/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: cellranger_vdj
+description: Module to use Cell Ranger's pipelines analyze sequencing data produced from Chromium Single Cell Immune Profiling.
+keywords:
+  - align
+  - vdj
+  - reference
+  - immunoprofiling
+  - single-cell
+  - cellranger
+tools:
+  - cellranger:
+      description: Cell Ranger processes data from 10X Genomics Chromium kits. `cellranger vdj` takes FASTQ files from `cellranger mkfastq` or `bcl2fastq` for V(D)J libraries and performs sequence assembly and paired clonotype calling. It uses the Chromium cellular barcodes and UMIs to assemble V(D)J transcripts per cell. Clonotypes and CDR3 sequences are output as a `.vloupe` file which can be loaded into Loupe V(D)J Browser.
+      homepage: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/what-is-cell-ranger
+      documentation: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/tutorial/tutorial-vdj
+      tool_dev_url: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/tutorial/tutorial-vdj
+      licence: 10x Genomics EULA
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - reference:
+      type: directory
+      description: Folder containing all the reference indices needed by Cell Ranger
+output:
+  - outs:
+      type: file
+      description: Files containing the outputs of Cell Ranger
+      pattern: "${meta.id}/outs/*"
```

Comment:
This isn't so informative as it's not hte asctual files that get output. Why not something like `*.{porefix1,prefix2,prefix3}` etc.?

Reply:
a reasonable compromise is to document whatever 10X thinks is an [average case output file set](https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/using/vdj#success)

Reply:
ok it occurred to me that the `meta.yml` should match the code in the module

the code output is two lines:
```
tuple val(meta), path("${meta.id}/outs/*"), emit: outs
path "versions.yml" , emit: versions
```
and tabulating all possible consistent + optional outputs here doesn't seem wise

I'm going to punt and add a note to check official 10X docs for a full output file list

will do for both `cellranger count` and `cellranger vdj` 👍🏼

Reply:
Fair enough!